src/views/pages/oss-cdn.html

```diff
@@ -55,7 +59,11 @@
 						{{/partial}}
 					</c-proxy-head>
 
-					<c-proxy-request-bandwidth-stats name="ghost" period="month" bandwidthUnit="GB"></c-proxy-request-bandwidth-stats>
+					<c-proxy-request-bandwidth-stats
+						name="ghost"
+						periodData="{periodValue: 'month', periodText: 'month', periodType: 's-month'}"
```

Comment:
Do we actually need to pass around the period data like this? I would say this should stay as `period=month` everywhere, and only in the period dropdown can you add the text based on the period value.

Reply:
Yes, because we use this component also on the _oss-cnd-project page
![2023-02-10_20-17](https://user-images.githubusercontent.com/13244319/218166913-356d191f-0f3f-4179-9a49-8c962432b1a1.png)